JakinChan200/CS180_Roald
# Added CRUD functionality for user-submitted videos using persistence in JSON and CSV formats.

## Overview
- Added /api/videos endpoint with CRUD functionality for user-submitted videos.
- GET /api/videos reads all user-submitted videos in the files.
- GET /api/videos/:id reads the user-submitted video(s) matching the given videoID.
- POST /api/videos/ creates a user-submitted video.
- PUT /api/videos/:id updates the user-submitted video matching the given videoID.
- DELETE /api/videos/:id deletes the user-submitted video matching the given videoID.
- Videos are stored in two formats, JSON and CSV.

## Materials
- #26

## Pictures
Provide a picture or video of your changes if applicable.

## Manual Testing Guide
Using Postman or cURL (or some method of using HTTP methods), visit the /api/videos endpoint.
- GET /api/videos reads all user-submitted videos in the files.
- GET /api/videos/:id reads the user-submitted video(s) matching the given videoID.
- POST /api/videos/ creates a user-submitted video.
- PUT /api/videos/:id updates the user-submitted video matching the given videoID.
- DELETE /api/videos/:id deletes the user-submitted video matching the given videoID.

Pull requests should receive approval before merge.
